client/state/stats/lists/reducer.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+import merge from 'lodash/merge';
+
+/**
+ * Internal dependencies
+ */
+import { isValidStateWithSchema } from 'state/utils';
+import { getSerializedStatsQuery } from './utils';
+import { items as itemSchema } from './schema';
```

Comment:
Elsewhere, I've seen a tendency toward simply exporting the schema with the `Schema` prefix to avoid needing to rename, i.e. `import { itemsSchema } from './schema';` . Doesn't appear to be much of a downside to doing so since the named export is equally meaningful as I see it.